frontend/src/router.js

```diff
@@ -518,25 +533,26 @@ export function campRoute(camp, subroute = 'dashboard', query = {}) {
 
 /**
  * @param camp
- * @param materialList
+ * @param materialListOrRoute { '/all', '/lists', object }
  * @param query
  */
-export function materialListRoute(camp, materialList = false, query = {}) {
-  if (materialList === false) {
+export function materialListRoute(camp, materialListOrRoute = '/all', query = {}) {
+  if (camp._meta.loading) return {}
+  if (typeof materialListOrRoute === 'string') {
     return {
-      name: 'material/overview',
+      name: `material${materialListOrRoute}`,
       params: { campId: camp.id, campTitle: slugify(camp.title) },
       query,
     }
   }
```

Comment:
`/all` and `/lists` should probably move to campRoute and have a route name prefix "camp/material/$1"